packages/material-ui/src/PaginationItem/PaginationItem.js

```diff
@@ -15,24 +14,22 @@ import NavigateBeforeIcon from '../internal/svg-icons/NavigateBefore';
 import NavigateNextIcon from '../internal/svg-icons/NavigateNext';
 import experimentalStyled from '../styles/experimentalStyled';
 
-const overridesResolver = (props, styles) => {
+const rootOverridesResolver = (props, styles) => {
   const { styleProps } = props;
 
-  return deepmerge(
-    {
-      ...styles[styleProps.variant],
-      ...styles[`size${capitalize(styleProps.size)}`],
-      ...(styleProps.variant === 'text' && styles[`text${capitalize(styleProps.color)}`]),
-      ...(styleProps.variant === 'outlined' && styles[`outlined${capitalize(styleProps.color)}`]),
-      ...(styleProps.shape === 'rounded' && styles.rounded),
-      [`&.${paginationItemClasses.ellipsis}`]: styles.ellipsis,
-      [`&.${paginationItemClasses.previousLast}`]: styles.previousLast,
```

Comment:
`previousLast` was wrong it didn't exist. Also no need for bumping specificity here.